src/components/Editor.js

```diff
@@ -25,7 +25,7 @@ const propTypes = {
   onTab: PropTypes.func,
   onUpArrow: PropTypes.func,
   onDownArrow: PropTypes.func,
-  readOnly: PropTypes.bool
```

Comment:
nit: i happen to like trailing commas but i didn't do it elsewhere in this project